### PR TITLE
Fix .bzl visibility warnings

### PR DESCRIPTION
**Pull request by vladmos**
The warning didn't work correctly with relative file paths.

### CODE REVIEW

**Comment by vladmos** [comment]
That's Windows- (case-insensitive) and MacOS- (different interface of `sed`) specific in the shell test. I had to update the shell test because the unit test for the warning didn't catch everything, i.e. the location of the WORKSPACE file is supposed to be detected not by the warning function but by the main buildifier binary.

I'll fix the test soon.

File: buildifier/utils/utils.go [hunk 1]
```diff
@@ -77,9 +77,22 @@ func GetParser(inputType string) func(filename string, data []byte) (*build.File
 	}
 }
 
+// hasWorkspaceFile checks whether a given directory contains a file called
+// WORKSPACE or WORKSPACE.bazel. The contents are not checked
```
Comment (laurentlb) [inline buildifier/utils/utils.go]: Remove "The contents are not checked".

**Comment by vladmos** [comment]
The tests are passing now.

File: buildifier/utils/utils.go (hunk 1 above)
Comment (vladmos) [reply buildifier/utils/utils.go]: Done.

**Pull request merged**